LICENSE

```diff
@@ -1179,6 +1287,18 @@ Dependency licenses:
     If the Library as you received it specifies that a proxy can decide whether future versions of the GNU Lesser General Public License shall apply, that proxy's public statement of acceptance of any version is permanent authorization for you to choose that version for the Library.
     
 
+  GPL2 w/ CPE: |
```

Comment:
@nakomis We can't include this, I believe! See https://www.apache.org/legal/resolved.html#category-x

Is it just brought in by `jakarta.ws.rs.jakarta.ws.rs-api:2.1.5`? That seems to have two licenses: EPL 2.0 and GPL2 w/ CPE. I think we can choose between the two, so I presume we can remove this?

@nakomis Can you compare the other new licenses with the black-listed license types please?